test/general/TestItems.py

```diff
@@ -29,3 +30,17 @@ def testItemNameGroupHasValidItem(self):
                         with self.subTest(group_name, group_name=group_name):
                             for item in items:
                                 self.assertIn(item, world_type.item_name_to_id)
+
+    def testItemCountGreaterEqualLocations(self):
+        for game_name, world_type in AutoWorldRegister.world_types.items():
+
+            if game_name not in {"Archipelago", "Final Fantasy"}:
+
```

Comment:
I think applying the change from alwaysintreble went sideways here :/ also incorporating berserker's comment
```suggestion
            if game_name not in {"Final Fantasy"}:
                continue
```
I ran it like that locally and it works

Reply:
Oh yeah everything after it would've needed to be indented. I just don't like using a continue for such a minor comparison lol